storage/innobase/os/os0file.cc

```diff
@@ -97,6 +97,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 #include <data0type.h>
 #endif /* UNIV_HOTBACKUP */
 
+/* Ignore posix_fadvise() on those platforms where it does not exist */
```

Comment:
This define is probably redundant. `POSIX_FADV_DONTNEED` will not be defined on Windows even if we cared about this platform.
Moreover, there are two more occurrences of `posix_fadvise()` in the code tree: in `row0log.cc` and `row0merge.cc`, where we do not redefine `posix_fadvise` as a macro.

Reply:
Right, we can safely remove this.